lib/babel-wrap.js

```diff
@@ -0,0 +1,16 @@
+var path = require('path')
+
+// remove ourselves from process.argv
+process.argv = process.argv.filter(function (arg) {
```

Comment:
This filter function is going to result in some false positives.  Probably it's better to just splice out the exact filename that we know it's going to be?